main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+# Introduction to Zoe
+
+
+**Note:** Before reading this, you should be familiar with ERTP basics. (**tyg todo**: Add link)
+
+
+This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide (**tyg todo**: Add link) has more complete and detailed information, including concepts not covered here. In particular, here we simplify Zoe examples such that there are always two parties in a basic trading/swap contract. We do not discuss more complicated contracts with many participants, such as auctions, or simpler contracts, such as our AutomaticRefund one party demo contract.
+
+
+## What is Zoe?
+
+
+*Zoe* is itself a long lived and trusted contract that serves as a framework which runs other contracts.
+
+
+## How do Zoe contracts work?
+
+
+To start using a contract, you need to *install* it on Zoe. This is putting the contract’s code into Zoe. Multiple contracts can be installed on Zoe, and each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code.
```

Comment:
> each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance.

Could easily be misunderstood to mean that there is an instance per active offer.

Reply:
Yeah, I see this. Will rephrase, although I could really do with a good word for what's going on from the time an instance is instantiated and "activated" by getting an offer through when the contract instance finishes and everyone gets whatever they get out of Zoe escrow. "Contract (or contract instance) execution" doesn't seem to be it. Any suggestions?